src/lambda/snowplow/userEventHandler.ts

```diff
@@ -0,0 +1,145 @@
+import { buildSelfDescribingEvent } from '@snowplow/node-tracker';
+import { SelfDescribingJson } from '@snowplow/tracker-core';
+import { EventType, SnowplowEventMap, UserEventPayloadSnowplow } from './types';
+import { Account, ApiUser, ObjectUpdate, User } from './types';
+import config from '../config';
+import { EventHandler } from './EventHandler';
+import { tracker } from './tracker';
+
+type ObjectUpdateEvent = Omit<SelfDescribingJson, 'data'> & {
+  data: ObjectUpdate;
+};
+
+type AccountContext = Omit<SelfDescribingJson, 'data'> & {
+  data: Account;
+};
+
+type UserContext = Omit<SelfDescribingJson, 'data'> & {
+  data: User;
+};
+
+type ApiUserContext = Omit<SelfDescribingJson, 'data'> & {
+  data: ApiUser;
+};
+
+/**
+ * This class MUST be initialized using the SnowplowHandler.init() method.
+ * This is done to ensure event handlers adhere to the EventHandlerInterface.
+ */
```

Comment:
This comment maybe came from somewhere else?

Reply:
fixed